Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Commands/Commands.cs

```diff
@@ -103,7 +104,7 @@ public static Command CopyActionMapSelection()
             return (in InputActionsEditorState state) =>
             {
                 CopyPasteHelper.CopyActionMap(state);
-                return state;
+                return state.With(cutElements: new List<CutElement>());
```

Comment:
Wouldn't it make sense to keep cutElements set to null when nothing is cut? This way it will allocate memory on the heap on every invocation.

Reply:
solved in [89d0280](https://github.com/Unity-Technologies/InputSystem/pull/1858/commits/89d02803b78db9be0a828b1156da7b1337e89953)